### ISSUE TITLE
Cannot click the login button on startup

### ISSUE THREAD

**Issue by Flambe** [marcus604/LastFMDuplicateDeleter#1]
```bash
DevTools listening on ws://127.0.0.1:61850/devtools/browser/db25fb48-0a3b-4033-a6c9-ff75a6ec1279
Traceback (most recent call last):
  File "C:\Users\steve\tmp\LastFMDuplicateDeleter\Main.py", line 289, in <module>
    main()
  File "C:\Users\steve\tmp\LastFMDuplicateDeleter\Main.py", line 158, in main
    signIn(browser, userConfig["username"], password)
  File "C:\Users\steve\tmp\LastFMDuplicateDeleter\Main.py", line 72, in signIn
    browser.find_element_by_xpath("//div[@class='form-submit']/button[@class='btn-primary']").click()
  File "C:\Users\steve\.virtualenvs\LastFMDuplicateDeleter-IxISXhAb\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
    self._execute(Command.CLICK_ELEMENT)
  File "C:\Users\steve\.virtualenvs\LastFMDuplicateDeleter-IxISXhAb\lib\site-packages\selenium\webdriver\remote\webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "C:\Users\steve\.virtualenvs\LastFMDuplicateDeleter-IxISXhAb\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\steve\.virtualenvs\LastFMDuplicateDeleter-IxISXhAb\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button type="submit" name="submit" class="btn-primary">...</button> is not clickable at point (825, 443). Other element would receive the click: <div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645"></div>
  (Session info: headless chrome=91.0.4472.124)
```

just trying to do a dry run on python 3.9.6. It looks like the UI may have changed?

**Comment by marcus604** [comment]
Working on my end here

Can you tell me which version of 

- Operating System
- Selenium chrome driver
- Chrome
- pipenv

Theres a chance last.fm is doing A/B testing and you're getting served a slightly different version of the site but hopefully not.

**Comment by Flambe** [comment]
- Operating System
Windows 10 Pro 21390.2025
- Selenium chrome driver
91.0.4472.101
- Chrome
91.0.4472.124
- pipenv
2021.5.29

Just tried again and I'm getting the same error. If it is A/B, hopefully it'll switch back soon

**Comment by marcus604** [comment]
So I've matched my environment identically to yours and cannot replicate the issue. 

Looking more closely at your logs the class that it would end up clicking is

 `<div class="onetrust-pc-dark-filter` 

This looks like a privacy piece of software called OneTrust, do you have any software from them installed, are you running this on a work machine that might have something like this on it.

If you can try with this software disabled as it looks like its injecting elements into the page which is whats causing this.

If you dont think you have this software installed, can you try replacing the Main.py with the version attached, only difference is you'll see the browser open and theres a delay between when it fills in the username and password, if you can screenshot what you see before it tries to click on the login button.

[Main.zip](https://github.com/marcus604/LastFMDuplicateDeleter/files/6781117/Main.zip)

**Comment by Flambe** [comment]
Looks like it's a cookies popup causing this. One Trust is not something I have installed (on my linux box at the moment and still getting the same issue)

![image](https://user-images.githubusercontent.com/2080851/124889632-9246ac80-dfd7-11eb-95cc-179ec568426d.png)

**Comment by marcus604** [comment]
Ok so this should be easy to fix, but even with a fresh Windows 10 VM I still cant replicate this popup, so I thought it'd be location based. Tried a lot of locations (Germany, multiple US, France, UK, Australia) with a VPN and multiple browsers both incognito and not and still no luck on getting this pop-up. Not sure what requirements are required for it are at this point. 

Can you use this new version of Main.py, only thing I've changed is the time it waits to try and click, while its waiting when it opens, without clicking on anything open the dev tools and get the entire HTML thats produced from the page, since its just the one button press I should be able to get it working without being able to replicate it on my end.

[Main.py.zip](https://github.com/marcus604/LastFMDuplicateDeleter/files/6788014/Main.py.zip)

**Comment by Flambe** [comment]
Copied it from the dev tools so hopefully it's not messed up.

[index.zip](https://github.com/marcus604/LastFMDuplicateDeleter/files/6790305/index.zip)

**Comment by marcus604** [comment]
Can you try the [iss01 branch version](https://github.com/marcus604/LastFMDuplicateDeleter/tree/iss01)

I've left it currently not headless so let me know if you actually see the popup and it handles it correctly. I still can't get the popup on last.fm myself but intermittently the button id is present but not viewable for me.

**Comment by Flambe** [comment]
@marcus604 the class was slightly off, but I changed it to the one in this MR and it works now! Takes a second for the popup to animate away, but logging in works fine now :) https://github.com/marcus604/LastFMDuplicateDeleter/pull/3_

**Comment by marcus604** [comment]
Thanks so much for PR! 

Fixed with #3

**Comment by Flambe** [comment]
Thanks for the quick fix! :)
